tests/test_class_import.py

```diff
@@ -134,12 +134,15 @@ def test_fake_strftime_function():
     assert fake_strftime_function() == '2012'
 
 
-@freeze_time("2022-01-01 12:00:00")
+@freeze_time("2022-01-01")
 def test_fake_time_function_as_class_attribute():
+    local_time = datetime.datetime(2022, 1, 1)
+    utc_time = local_time - datetime.timedelta(seconds=time.timezone)
+    expected_timestamp = time.mktime(utc_time.timetuple())
```

Comment:
This part is duplicated from `test_import_time`.
I can extract it to `_date_to_time(year: int, mont: int, day: int)` at the end of the module, wdyt ?

Reply:
:heavy_check_mark: at 9d2b92f7faedec016ab17b088ebf330024ddf15f